src/components/panel/editForm/Panel.edit.display.js

```diff
@@ -123,6 +123,16 @@ export default [
           cancel: true,
           next: true
         },
+      },
+      {
+        weight: 50,
+        label: 'Scroll to the top when open',
```

Comment:
I believe we now want this to be the default. Please remove the option, and then just make it the default. Also please write a test for this.